week-3/todo-list/script.js

```diff
@@ -1,25 +1,96 @@
+// function populateTodoList(todos) {
+//   let list = document.getElementById("todo-list");
+//   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+// }
+
+// // These are the same todos that currently display in the HTML
+// // You will want to remove the ones in the current HTML after you have created them using JavaScript
+// let todos = [
+//   { task: "Wash the dishes", completed: false },
+//   { task: "Do the shopping", completed: false },
+// ];
+
+// populateTodoList(todos);
```

Comment:
This code (llines 8 to 13) shouldnt be deleted.  Line 8 initialises a list of the todos and line 13 initialises the page with the default list of todo items (wash the dishes and do the shopping).  It is important you keep the `todos` list because each new item should be added to the Javascript list AND the HTML list.  Then you have a list of todo items in your code that you can manage and the HTML page is just a display of those.  As the comment says on line 3, each todo item should have a `completed` and a `delete` button.  Your list does not have those buttons.